### ISSUE TITLE
[CoE Starter Kit - BUG] Sync Template v3 (PVA) - Create Bot action failed: "The length of substring can't be longer than '6' which is the length of the source string." on the Dataverse action "Create Bot".

### ISSUE THREAD

**Issue by thenchoz** [microsoft/coe-starter-kit#6000]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

![CoE_failed](https://github.com/microsoft/coe-starter-kit/assets/28315645/a3acd205-228e-40a6-b6cd-cdaa2e4ab9db)

Hi,

The flow failed on an existing bot which has an active owner with error "The length of substring can't be longer than '6' which is the length of the source string." on the Dataverse action "Create Bot".

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.9

### What app or flow are you having the issue with?

Admin | Sync Template v3 (PVA)

### What method are you using to get inventory and telemetry?

None

**Comment by Jenefer-Monroe** [comment]
Can you please go to the **_Parse Bot Json_** action for this failing bot, and get the _componentstate_label
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/9dbff49f-686a-4f92-898b-04c3a808205a)

**Comment by thenchozGRE** [comment]
Here is the value: "_componentstate_label": "Publié"

**Comment by thenchozGRE** [comment]
Hello,

Thank you for your quick answer. The CoE environment is already in English, as all production environment, but not the dev and default ones. And sadly this is not an option, as many person from our organisation don't speak English at all. We can either have the advantage of the Low Code with a lot of developer and loads of app, or the environment in English, but not both.

How would it be possible to overturn this issue without having to disable that flow?

**Comment by Jenefer-Monroe** [comment]
I think I just have the wrong pictures there. Sorry about that, removing them. You should be able to add the language pack without having it be the display language.
We can also explore making a change for this particular flow but cannot comment on when that would surface in priority.

**Comment by Jenefer-Monroe** [comment]
ok we got curious about why we put label in there and now cant recall what the reason was. Given it was 3 years ago we have decided to assume the reason no longer applies as we have confirmed that this does work: 

instead of 
if(equals(substring(body('Parse_Bot_JSON')?['_componentstate_label'], 0, 7), 'Deleted'), true, false)

we have put 
if(equals(body('Parse_Bot_JSON')?['componentstate'], 2), true, false)

And ran a test case against English, French and Japanese. So we have local fixed this for August.
